Coopersbranch

CooperW12 opened the pull request.

Commit:
CollectiveTreeCode

Commit:
CollectiveTreeCode

Team1710CTO commented:
Make sure to test it and to fix it before making the pull request

Team1710CTO commented on MyClass.java:
```diff
@@ -4,11 +4,26 @@
     getRows(); 
     row();
     stump();
+>>>>>>> bee68808de44320c3737a7c9146f8c7fa0194c59
```
Is this meant to be a comment?

Team1710CTO commented on MyClass.java:
```diff
@@ -54,15 +73,35 @@ public static void row(int spaces, int stars){
     
     
   public static void stump(int width, int rows){ //Varibles can be changed; Need varible for width
+<<<<<<< HEAD
```
I don't believe this is along with all the other lines like it E.G(=======) are executable,

Team1710CTO commented on MyClass.java:
```diff
@@ -4,11 +4,26 @@
 
    
     public static void main(String args[]) {
+<<<<<<< HEAD
```
see comments below

Team1710CTO commented on MyClass.java:
```diff
@@ -75,3 +114,4 @@ public static void row(int spaces, int stars){
 }
 }
 
+>>>>>>> bee68808de44320c3737a7c9146f8c7fa0194c59
```
This line wont compile along with others like it. You'll need to make deletions of lines like this one for it to run properly.

CooperW12 committed:
Update MyClass.java

Fixed things

CooperW12 commented:
I removed all of that unnecessary text. I think it was from the way I copied it over from the compiler. I also fixed up a few spacing and code issues.

Team1710CTO commented on MyClass.java:
```diff
@@ -4,23 +4,29 @@
+    System.out.println("How many rows do you want in the tree height? ==>");
+    int rows = rowScanner.nextInt();
+    
+int i = 0;
```
Indentation

CooperW12 committed:
Update MyClass.java
